src/index.html

```diff
@@ -13,6 +13,40 @@
     />
```

Comment:
It seems like you are linking a SCSS file directly into your HTML. Web browsers do not understand SCSS code; they require CSS. You should compile your SCSS to CSS and link to the CSS file instead.